jest.config.cjs

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  coverageDirectory: '<rootDir>/test-results/coverage',
+  collectCoverageFrom: ['src/**/*.{ts,tsx}', '!src/**/*-stories.tsx', '!src/**/index.ts'],
+  coverageReporters: ['lcovonly', 'text'],
+  moduleFileExtensions: ['ts', 'tsx', 'js'],
+  moduleNameMapper: {
+    '^.+\\.(md|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
+      '<rootDir>/config/jest/FileStub.js',
+    '^.+\\.css$': '<rootDir>/config/jest/CSSStub.js',
+  },
+  setupFiles: ['<rootDir>/config/jest/SetupTestEnvironment.ts'],
+  setupFilesAfterEnv: [
+    '<rootDir>/config/jest/SetupReactTestingLibrary.ts',
+    '<rootDir>/config/jest/SetupJestAxe.ts',
+  ],
+  snapshotSerializers: ['@emotion/jest/serializer'],
+  testEnvironment: 'jsdom',
+  testPathIgnorePatterns: [
+    '<rootDir>/config',
+    '<rootDir>/design-system',
+    '<rootDir>/node_modules',
+    '<rootDir>/scripts',
+  ],
+  testRegex: '(/__tests__/.*|\\-test)\\.(ts|tsx)$',
+  transform: {
+    '^.+\\.(t|j)sx?$': [
+      '@swc/jest',
```

Comment:
Can't we leverage vite for the transform ? To avoid adding more technologies than needed in the repo

Reply:
I'm not sure, but you can try!

Reply:
Just had a quick look and it seems like vite integration with jest isn't so great, let's keep it like this.